src/frontend/web_application/src/services/openpgp-keychain-repository/local-storage-helper.js

```diff
@@ -0,0 +1,20 @@
+export function findAll(namespace) {
+  return Object.keys(window.localStorage)
```

Comment:
localStorage is not always availalble, you'll probably be safer testing for it's support first, or you'll end up with a TypeError ;)